pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -95,22 +95,25 @@ func (r *Builder) Import(vmRef ref.Ref, object *vmio.VirtualMachineImportSpec) (
 	if pErr != nil {
 		err = liberr.New(
 			fmt.Sprintf(
-				"VM %s lookup failed: %s",
+				"VM %s (%s) lookup failed: %s",
+				vm.Name,
 				vmRef.String(),
```

Comment:
We use the Ref.String() everywhere for both error and logging.  Let's change it to provide better information.
Perhaps:
```
//
// String representation.
// Format: (<type>) id: <id> name: '<name>'
func (r *Ref) String() (s string) {
	if r.Type != "" {
		s = fmt.Sprintf(
			"(%s)",
			r.Type)
	}
	s = fmt.Sprintf(
		"%s id:%s name:'%s' ",
		s,
		r.ID,
		r.Name)
	
	return
}
```